src/components/ContentBook.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ContentBook = (props) => {
+  const { title, author } = props;
+  return (
+    <ul>
+      <li>
+        <h3>{title}</h3>
+        <p>{author}</p>
+        <button type="button">Delete</button>
+      </li>
+    </ul>
```

Comment:
- [ ] Great job working on this section! 💯 However, keep in mind that you have a duplicate `<ul>` tag here. This component represents a single book of the list, so it makes sense to use just the `<li>` as the container. Its parent is wrapping all instances of ContentBook inside the `<ul>` already. 🤓 🔧 

```suggestion
    
      <li>
        <h3>{title}</h3>
        <p>{author}</p>
        <button type="button">Delete</button>
      </li>
    
```